examples/llamacpp_example.py

```diff
@@ -31,7 +31,7 @@ class Character(BaseModel):
 
 if __name__ == "__main__":
     # Download model from https://huggingface.co/TheBloke/phi-2-GGUF
-    model = outlines.models.llamacpp("./phi-2.Q3_K_M.gguf", device="cpu")
+    model = outlines.models.llamacpp("./phi-2.Q4_K_M.gguf", device="cpu")
```

Comment:
Is the `device` argument here doing anything?

Reply:
Nope. This is a left-over from the older implementation. Will remove it.

Reply:
Should we still let the user specify?

Reply:
Ok for me

Reply:
I just realized that I misunderstood the comment (don't read comments late at nigh :smile:). The device handling can be controlled via kwargs. E.g. `n_gpu_layers` and `main_gpu`. If this has an effect is based on if you built llama-cpp-python with cublas (or metal etc.) support.